src/Parser/VectorTileParser.js

```diff
@@ -209,6 +224,10 @@ export default {
      */
     parse(file, options) {
         options = deprecatedParsingOptionsToNewOne(options);
-        return Promise.resolve(readPBF(file, options));
+        if (Array.isArray(file)) {
+            return Promise.resolve(multiplePBF(file, options));
+        } else {
+            return Promise.resolve(readPBF(file, options));
+        }
```

Comment:
If we think about the name and utility of this function, it is writen to parse a specifique file and thus it shouldn't be able to deal with a list of files.